src/main/java/it/gov/pagopa/pu/bff/connector/process_executions/client/ExportFileSearchClient.java

```diff
@@ -0,0 +1,43 @@
+package it.gov.pagopa.pu.bff.connector.process_executions.client;
+
+import it.gov.pagopa.pu.bff.connector.process_executions.config.ProcessExecutionsApisHolder;
+import it.gov.pagopa.pu.bff.dto.ExportFileFiltersDTO;
+import it.gov.pagopa.pu.bff.util.PageUtils;
+import it.gov.pagopa.pu.processexecutions.dto.generated.PagedModelExportFile;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+public class ExportFileSearchClient {
+
+  private final ProcessExecutionsApisHolder processExecutionsApisHolder;
+
+  public ExportFileSearchClient(
+    ProcessExecutionsApisHolder processExecutionsApisHolder) {
+    this.processExecutionsApisHolder = processExecutionsApisHolder;
+  }
+
+  public PagedModelExportFile getExportFiles(
+    ExportFileFiltersDTO exportFileFilters, String operatorExternalId,
+    Pageable pageable, String accessToken) {
+    String status = exportFileFilters.getStatus() != null ?
+      exportFileFilters.getStatus().name()
+      : "";
```

Comment:
not empty string! null! otherwise it will be send, giving error due to the invalid value!
ps: use the ternary operator directly on the method invoke, it would be more readable there (it's not used anywere else)